index.js

```diff
@@ -33,6 +33,17 @@ const LEDGER_SERVERS = {
   }
 }
 
+const PUBLISHER_HOSTS = {
+  'staging': {
+    v1: 'bat-publishers-staging-2.herokuapp.com',
+    v2: 'bat-publishers-staging-2.herokuapp.com'
+  },
+  'production': {
+    v1: 'bat-publishers-staging-2.herokuapp.com',
+    v2: 'bat-publishers-staging-2.herokuapp.com'
+  }
+}
+
```

Comment:
These are temporary right? Is this blocked on these being updated?

Reply:
Yes temporary for now. Will update once staging/prod endpoints are finalized and live.